Gemfile

```diff
@@ -9,12 +9,13 @@ gem 'puma', '~> 4.1'
 gem 'bootsnap', '>= 1.4.2', require: false
 
 group :development, :test do
-  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem 'rspec-rails'
   gem 'shoulda-matchers'
   gem 'factory_bot_rails'
   gem 'pry-rails'
   gem 'coveralls', require: false
+  gem 'rest-client'
```

Comment:
Haha.. Noooo. `rest-client` is used in production. `webmock` is not.

Reply:
Oh, we thought it was regarding both gems added, our bad! Now rest-client is back where it belongs!